pkg/stream/producer.go

```diff
@@ -96,8 +129,17 @@ func (p *producer) WriteOne(ctx context.Context, model interface{}, attributeSet
 	}
 
 	err = p.output.WriteOne(ctx, msg)
+	if shouldRetry(err, p.retryEnabled) {
+		retryErr := p.retryDaemon.RetryOne(ctx, msg)
+		if retryErr == nil {
+			return nil
+		}
+
+		return fmt.Errorf("can not write messages to output: %w", errors.Join(err, retryErr))
+	}
+
 	if err != nil {
-		return fmt.Errorf("can not write msg to output: %w", err)
+		return fmt.Errorf("can not write messages to output: %w", err)
```

Comment:
This log is in the WriteOne function I suppose it is a single message that is failing